paddle/fluid/platform/flags.cc

```diff
@@ -121,6 +121,9 @@ PADDLE_DEFINE_EXPORTED_string(
     "If proveided, it will be passed to aclInit().");
 PADDLE_DEFINE_EXPORTED_int32(min_loss_scaling, 1,
                              "set minmum loss scaling value!");
+PADDLE_DEFINE_EXPORTED_string(
```

Comment:
npu_precision_mod -> npu_precision_mode.
Better default  "", and please list all candidates mode.

Reply:
done thanks.